File: Msys_Programs/MSys_Projects/LeetCode/GoLang/Palindrome_No.go
package main

import "fmt"

func isPalindrome(x int) bool {
	if x < 0 {
		return false
	}
	t := x
	r := 0
	for x > 0 {
		r = r*10 + x%10
		x /= 10
	}
	return t == r
}

func main() {
	fmt.Println(isPalindrome(101))
	fmt.Println(isPalindrome(121))
	fmt.Println(isPalindrome(102))
}

File: Msys_Programs/MSys_Projects/LeetCode/Python/Palindrome_No.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if(x>=0):
            return x == int(str(x)[::-1])
        return False

P = Solution()
print(P.isPalindrome(233))
print(P.isPalindrome(212))
print(P.isPalindrome(21212))
print(P.isPalindrome(320023))